integration/sawtooth_integration/docker/test_poet_liveness.yaml

```diff
@@ -169,12 +161,8 @@ services:
       PYTHONPATH: "/project/sawtooth-core/consensus/poet/common:\
         /project/sawtooth-core/consensus/poet/simulator:\
         /project/sawtooth-core/consensus/poet/core:\
-<<<<<<< HEAD
         /project/sawtooth-core/sdk/python:\
-        /project/sawtooth-core/families/block_info"
-=======
         /project/sawtooth-core/families/block_info/python"
->>>>>>> 1c80e9a2... Add block-info tp in rust
```

Comment:
As mentioned in the previous comment this fixes should be added to the previous commit.